scripts/feature_tests/upgrade/workers_upgrade/1cp_3w_bootDiskImage_scaleInWorkers_upgrade_both.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -x
+
+# shellcheck disable=SC1091
+source ../common.sh
+
+echo '' >~/.ssh/known_hosts
+
+start_logging "${1}"
+
+# Old name does not matter
+export new_wr_metal3MachineTemplate_name="test1-new-workers-image"
+export new_cp_metal3MachineTemplate_name="test1-new-controlplane-image"
+
+set_number_of_master_node_replicas 1
+set_number_of_worker_node_replicas 3
+
+provision_controlplane_node
+wait_for_ctrlplane_provisioning_start
+
+CP_NODE=$(kubectl get bmh -n metal3 | grep control | grep -v ready | cut -f1 -d' ')
+echo "BareMetalHost ${CP_NODE} is in provisioning or provisioned state"
+CP_NODE_IP=$(sudo virsh net-dhcp-leases baremetal | grep "${CP_NODE}" | awk '{{print $5}}' | cut -f1 -d'/')
+wait_for_ctrlplane_provisioning_complete "${CP_NODE}" "${CP_NODE_IP}" "original controlplane node"
+
+# apply CNI
+ssh -o PasswordAuthentication=no -o "StrictHostKeyChecking no" "${UPGRADE_USER}@${CP_NODE_IP}" -- kubectl apply -f https://docs.projectcalico.org/manifests/calico.yaml
+
+provision_worker_node
+wait_for_worker_provisioning_start
+
+echo "Create a new metal3MachineTemplate with new node image for worker nodes"
+wr_Metal3MachineTemplate_OUTPUT_FILE="/tmp/wr_new_image.yaml"
+
+CLUSTER_UID=$(kubectl get clusters -n metal3 test1 -o json | jq '.metadata.uid' | cut -f2 -d\")
+IMG_CHKSUM=$(curl -s https://cloud-images.ubuntu.com/bionic/current/MD5SUMS | grep bionic-server-cloudimg-amd64.img | cut -f1 -d' ')
```

Comment:
please refer to my comment wrt the checksum

Reply:
replaced with local checksum value (also the image is local now)